tests/integration/update.bats

```diff
@@ -557,6 +557,33 @@ EOF
 	check_systemd_value "AllowedMemoryNodes" 1
 }
 
+@test "update cpuset cpus range via v2 unified map" {
+	# This test assumes systemd >= v244
+	[ $EUID -ne 0 ] && requires rootless_cgroup
+	requires cgroups_v2 more_than_8_core cgroups_cpuset
```

Comment:
Since this test uses `check_systemd_value`, it is sort of useless without systemd.

Please add `require systemd`.

Reply:
Thanks for pointing out my mistake. I have added `requires systemd`.